src/controllers/groupController.js

```diff
@@ -0,0 +1,119 @@
+import { save, getById, update, getAllGroups, remove, addUsersToGroup } from '../services/GroupService';
+import { v4 as uuidv4 } from 'uuid';
+import { notFoundEntity, statusNotFound } from '../util/constant';
+import { omitPassword } from '../util/util';
+
+export const saveGroup = async (request, response, next) => {
+    let group = { id: uuidv4(), ...request.body };
+    response.locals.log = `saveGroup ${JSON.stringify(group)}`;
+    try {
+        group = await save(group);
+        response.locals.status = 201;
+        response.locals.send = group;
+        next();
+    } catch (err) {
+        err.message = `saveGroup failed: ${JSON.stringify(group)}, message: ${err.message}`;
+        return next(err);
+    }
+};
+
+export const getGroup = async (request, response, next) => {
+    const id = request.params.id;
+    response.locals.log = `getGroup by id: ${id}`;
+    try {
+        const group = await getById(id);
+        if (group) {
+            response.locals.status = 200;
+            response.locals.send = group;
+            next();
```

Comment:
you can take away next() call from if/else? because it will be called anyway in each branch. Change the same in other methods